Unable to install topicmodels package

I am having trouble installing the `topicmodels` package. It looks like there is a broken dependency with the `gsl` library. I have tried installing `gsl` through `apt-get`, but it seems the headers are getting installed someplace where R can't find them.

```
* installing *source* package ‘topicmodels’ ...
** package ‘topicmodels’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
gcc -I"/usr/local/lib/R/include" -DNDEBUG   -I/usr/local/include  -fpic  "-I/usr/include/gsl/" -c cokus.c -o cokus.o
gcc -I"/usr/local/lib/R/include" -DNDEBUG   -I/usr/local/include  -fpic  "-I/usr/include/gsl/" -c common.c -o common.o
gcc -I"/usr/local/lib/R/include" -DNDEBUG   -I/usr/local/include  -fpic  "-I/usr/include/gsl/" -c ctm.c -o ctm.o
ctm.c:29:25: fatal error: gsl/gsl_rng.h: No such file or directory
 #include <gsl/gsl_rng.h>
                         ^
compilation terminated.
/usr/local/lib/R/etc/Makeconf:167: recipe for target 'ctm.o' failed
make: *** [ctm.o] Error 1
ERROR: compilation failed for package ‘topicmodels’
* removing ‘/usr/local/lib/R/site-library/topicmodels’

The downloaded source packages are in
	‘/tmp/RtmpNWaQuO/downloaded_packages’
Warning message:
In install.packages("topicmodels") :
  installation of package ‘topicmodels’ had non-zero exit status
```

Thank you.

--- a/package_installs.R
+++ b/package_installs.R
@@ -48,3 +48,6 @@ install.packages("prophet")
 # handled in this image's Dockerfile.
 install.packages("fftw")
 install.packages("seewave")
+
+# https://github.com/Kaggle/docker-rstats/issues/73
+install.packages("topicmodels")

--- a/tests/test_topicmodels.R
+++ b/tests/test_topicmodels.R
@@ -1,0 +1,22 @@
+context("topicmodels")
+
+# https://eight2late.wordpress.com/2015/09/29/a-gentle-introduction-to-topic-modeling-using-r/
+test_that("basic example", {
+	expect_error({
+		library(topicmodels)
+
+		#Set parameters for Gibbs sampling
+		burnin <- 4000
+		iter <- 2000
+		thin <- 500
+		seed <-list(2003,5,63,100001,765)
+		nstart <- 5
+		best <- TRUE
+
+		#Number of topics
+		k <- 5
+
+		#Run LDA using Gibbs sampling
+		ldaOut <-LDA(dtm,k, method=”Gibbs”, control=list(nstart=nstart, seed = seed, best=best, burnin = burnin, iter = iter, thin=thin))
+	}, NA) # expect no error to be thrown
+})

--- a/tests/test_topicmodels.R
+++ b/tests/test_topicmodels.R
@@ -17,6 +17,6 @@ test_that("basic example", {
 		k <- 5
 
 		#Run LDA using Gibbs sampling
-		ldaOut <-LDA(dtm,k, method=”Gibbs”, control=list(nstart=nstart, seed = seed, best=best, burnin = burnin, iter = iter, thin=thin))
+		ldaOut <-LDA(dtm,k, method="Gibbs", control=list(nstart=nstart, seed = seed, best=best, burnin = burnin, iter = iter, thin=thin))
 	}, NA) # expect no error to be thrown
 })

